conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/michelson/contracts/SmartContracts.scala

```diff
@@ -0,0 +1,91 @@
+package tech.cryptonomic.conseil.indexer.tezos.michelson.contracts
+
+import java.nio.charset.StandardCharsets
+import scorex.util.encode.{Base16 => Hex}
+import org.bouncycastle.jcajce.provider.digest.Blake2s.Blake2s128
+import cats.data.Chain
+import cats.implicits._
+
+/** Generic utilities related to smart contracts on tezos */
+object SmartContracts {
```

Comment:
To follow our naming convention so far, can we call this `SmartContractUtils`?

Reply:
sure

Reply:
First consider this: the code is inside a different package and module than the other *Utils.
The latter usually live in the `common` module under package `tech.cryptonomic.conseil.common.util`

This lives in package `tech.cryptonomic.conseil.indexer.tezos.michelson.contracts`
alongside
 - TokenContracts.scala
 - TNSContract.scala

Which in a sense makes the naming more consistent with the siblings